conductor/clarisse_utils.py

```diff
@@ -103,24 +103,24 @@ def do_export():
     scene_info = {"output_path": ""}
     for i in range(attrs.get_count()):
         # deduplicating
-        file = attrs[i].get_string()
-        if not os.path.isfile(file):
+        _file = attrs[i].get_string()
+        if not os.path.isfile(_file):
             #  Find the output path...
             if attrs[i].get_name() == "save_as":
-                scene_info['output_path'] = os.path.dirname(file)
+                scene_info['output_path'] = os.path.dirname(_file)
 
-            print("Skipping file %s" % file)
+            print("Skipping file %s" % _file)
             continue
         attr_list.append(attrs[i].get_full_name())
-        if not file in unique_files:
+        if not _file in unique_files:
             # de-windoify path
-            new_filename = os.path.abspath(file).replace("\\", '/').replace(':', '').replace('\\\\', '/')
+            new_filename = os.path.abspath(_file).replace("\\", '/').replace(':', '').replace('\\\\', '/')
             # getting the absolute path of the file
             new_filename = "$PDIR/" + new_filename

```

Comment:
While we're at it, `new_file_path` makes more sense; a file name is `foo.bar` whereas a file path is `/root/foo.bar`, which is what this variable contains.